pkg/trace/api/api.go

```diff
@@ -28,6 +28,8 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/tinylib/msgp/msgp"
```

Comment:
I think you got it backwards 😅 I need to write a wiki for the coding style, otherwise I'm just annoying. I promise I will do it soon.